cyberark/conjur
# Hero section scss cleanup

Closes [452](https://github.com/cyberark/conjur/issues/452)

#### What does this pull request do?
Reworks the template and css for the CTAs in the hero section. Visually, they look the same, but are built to use data files instead of editing the markup.
#### What background context can you provide?
We're past the initial push to get conjur.org live, and as we start to expand the site and its content, we're starting to reorganize and refactor some of the css.
#### Where should the reviewer start?
homepage
#### How should this be manually tested?
Make sure the CTAs in the hero section are visually consistent, and stack at smaller screen widths. Please also check in IE.
#### Screenshots (if appropriate)
![screenshot 2017-10-20 14 26 06](https://user-images.githubusercontent.com/123787/31836269-ad1808be-b5a2-11e7-8f71-3786b9fe84ae.png)
![screenshot 2017-10-20 14 26 20](https://user-images.githubusercontent.com/123787/31836276-b3ebc4fa-b5a2-11e7-8fcc-44cd9c55bdcb.png)

#### Link to build in Jenkins (if appropriate)
https://jenkins.conjur.net/job/cyberark--conjur/job/scss-cleanup-1/
#### Questions:
> Does this have automated Cucumber tests?
no

> Can we make a blog post, video, or animated GIF out of this?
no

> Is this explained in documentation?
no

> Does the knowledge base need an update?
no
